### PR TITLE
feat(nfts): add support for NFTs

### PR DESCRIPTION
**Pull request by tad3j**

### CODE REVIEW

File: packages/cli/src/types.d.ts
Comment (tinemlakar) [inline packages/cli/src/types.d.ts:1]: move interfaces to /modules/common/types.ts

File: packages/cli/tsconfig.json [hunk 1]
```diff
@@ -4,12 +4,15 @@
     "outDir": "./dist",
     "rootDir": "./src"
   },
+  "files": [
```
Comment (tinemlakar) [inline packages/cli/tsconfig.json:7]: remove this

File: packages/cli/src/types.d.ts
Comment (tinemlakar) [reply packages/cli/src/types.d.ts:1]: or even better, move everything from src/modules/common to src/lib to bi consistent with SDK package

File: packages/sdk/src/constants/nfts.ts
Comment (tinemlakar) [inline packages/sdk/src/constants/nfts.ts:1]: move this enum to /src/types/nfts.ts

File: packages/sdk/tsconfig.json [hunk 2]
```diff
@@ -4,12 +4,15 @@
     "rootDir": "./src",
     "outDir": "./dist"
   },
+  "files": [
```
Comment (tinemlakar) [inline packages/sdk/tsconfig.json:7]: remove this

**Review by tinemlakar — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -21,7 +21,6 @@ dist/
 
 # Maven
 target/
-dist/
```
[the remark this answers is not in the document]
Comment (tad3j) [reply .gitignore:24]: duplicate entry

**Pull request closed without merging**